fetch.bs

```diff
@@ -6030,7 +6152,8 @@ optional boolean <var>forceNewConnection</var> (default false), run these steps:
       buffer of up to 64 kibibytes and store a part of <var>request</var>'s <a for=request>body</a>
       in that buffer. If the user agent reads from <var>request</var>'s <a for=request>body</a>
       beyond that buffer's size and the user agent needs to resend <var>request</var>, then instead
-      return a <a>network error</a>.
+      return the result of running <a>HTTPS upgrade fallback</a> given <var>request</var> and
+      <var>network error</var>.
```

Comment:
This again is a streaming case. It won't be true for navigation.

Reply:
Removed this and the change above, but this leaves only step 7.1 of network fetch where fallback navigations are handled. If that's the only place where navigation failures can result in a network error, that's OK, but wanted to double check.

Reply:
That matches my analysis https://github.com/whatwg/fetch/pull/1655#discussion_r1390767516 so I think that's okay. Of course, I'm not infallible. 😅